bitnami/flux/values.yaml

```diff
@@ -2162,6 +2174,9 @@ imageAutomationController:
   ## @param imageAutomationController.args Override default container args (useful when using custom images)
   ##
   args: []
+  ## @param imageAutomationController.automountServiceAccountToken Mount Service Account token in pod
+  ##
+  automountServiceAccountToken: true
```

Comment:
shouldn't it be set to `false` instead?

Reply:
They require access to the K8s API so they need to mount the SA token